dask/dataframe/backends.py

```diff
@@ -45,6 +46,151 @@
 from dask.sizeof import SimpleSizeof, sizeof
 from dask.utils import is_arraylike, is_series_like, typename
 
+
+class DataFrameBackendEntrypoint(DaskBackendEntrypoint):
+    """Dask-DataFrame version of ``DaskBackendEntrypoint``
+
+    See Also
+    --------
+    PandasBackendEntrypoint
+    """
+
+    def __init__(self):
+        """Register data-directed dispatch functions"""
+        raise NotImplementedError
```

Comment:
>Ah, we need to potentially run registration in __init__.

Right. I'm attempting to "remind" future developers that compute-fuctions must be registered. Not sure if this is the best way to do that.

Reply:
Seems like this reminder isn't really that helpful, since it's possible to register at the module level, and that's what you end up doing below (but not without implementing a no-op `__init__`)...